src/browser-id3-writer.js

```diff
@@ -64,6 +64,17 @@ function getStringFrameSize(frameSize) {
     return headerSize + encodingSize + bomSize + frameUtf16Size;
 }
 
+function getUserStringFrameSize(descriptionSize, valueSize) {
+    const headerSize = 10;
+    const encodingSize = 1;
+    const bomSize = 2;
+    const descriptionUtf16Size = descriptionSize * 2;
+    const separatorSize = 2;
+    const valueUtf16Size = valueSize * 2;
+
+    return headerSize + encodingSize + bomSize + descriptionUtf16Size + separatorSize + valueUtf16Size;
```

Comment:
After **separator** and before **value** it should be another BOM for **value** like in USLT frame, isn't it?